doc/interfaces/tracking_flow.rst

```diff
@@ -0,0 +1,230 @@
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz data/stanford_hardi/pve_wm.nii.gz --out_dir "eudx_tracking_output"
+
+Deterministic Tracking
+**********************
+
+Deterministic maximum direction getter is the deterministic version of the 
+probabilistic direction getter. It can be used with the same local models and
+has the same parameters. Deterministic maximum fiber tracking follows the
+trajectory of the most probable pathway within the tracking constraint (e.g.
+max angle). In other words, it follows the direction with the highest
+probability from a distribution, as opposed to the probabilistic direction
+getter which draws the direction from the distribution. Therefore, the maximum
+deterministic direction getter is equivalent to the probabilistic direction
+getter returning always the maximum value of the distribution.
+
+Deterministic maximum fiber tracking is an alternative to EuDX deterministic
+tractography, and unlike EuDX, it does not follow the peaks of the local models,
+but uses the entire orientation distributions.
+
+We will first create the directory in which to save the output tractogram file
+(e.g.: ``det_tracking_output``)::
+
+    mkdir det_tracking_output
+
+Then, to perform the deterministic tracking we will run the ``dipy_track``
+command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz data/stanford_hardi/pve_wm.nii.gz --seed_density 2 --tracking_method "det" --out_dir "det_tracking_output"
+
+Probabilistic Tracking
+**********************
+
+Probabilistic fiber tracking is a way of reconstructing white matter connections
+using diffusion MR imaging. Like deterministic fiber tracking, the probabilistic
+approach follows the trajectory of a possible pathway step by step starting at a
+seed; however, unlike deterministic tracking, the tracking direction at each
+point along the path is chosen at random from a distribution. The distribution
+at each point is different and depends on the observed diffusion data at that
+point. The distribution of tracking directions at each point can be represented
+as a probability mass function (PMF) if the possible tracking directions are
+restricted to discrete numbers of well distributed points on a sphere.
+
+We will first create the directory in which to save the output tractogram file
+(e.g.: ``det_tracking_output``)::
+
+    mkdir det_tracking_output
+
+Then, to perform the probabilistic tracking we will run the ``dipy_track``
+command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz data/stanford_hardi/pve_wm.nii.gz --seed_density 2 --tracking_method "prob" --out_dir "det_tracking_output"
+
+Closest Peaks Tracking
+**********************
+
+Closest peaks fiber tracking is a type of deterministic tracking. Deterministic
+streamlines follow a predictable path through the data by selecting the same
+diffusion orientation when evaluating a point through the propagation process.
```

Comment:
Reading this again, I'd slightly modify this: "(...) by selecting the same diffusion orientation when evaluating the same point through the propagation process."

Reply:
Addressed in 74bf1c0